src/tests/JIT/HardwareIntrinsics/Arm/Shared/_SveMinimalUnaryOpTestTemplate.template

```diff
@@ -294,6 +297,17 @@ namespace JIT.HardwareIntrinsics.Arm
             ConditionalSelectScenario_FalseValue({RetVectorType}<{RetBaseType}>.AllBitsSet, _fld1, {RetVectorType}<{RetBaseType}>.Zero);
         }
 
+        public void RunScenario_AbsorbConditionalSelectOp()
+        {
+            TestLibrary.TestFramework.BeginScenario("RunScenario_AbsorbConditionalSelectOp - zero in falseValue");
+            var mask = Vector.As<{RetBaseType}, {Op1BaseType}>(_mask);
+            var zeroVec = {Op1VectorType}<{Op1BaseType}>.Zero;
+            var result = {Isa}.{Method}(Sve.ConditionalSelect(mask, _fld1, zeroVec));
```

Comment:
do we have coverage for `Sve.AddAcross(Sve.ConditionalSelect(AllOne, _fld1, zero))` or some more combination like that?

Reply:
We don't at the moment but happy to add if you suggest so 🙂 .

Reply:
worth adding few to support https://github.com/dotnet/runtime/pull/109218#discussion_r1837063846 scenario.

Reply:
Done.